src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,27 @@ title = "Releases"
 weight = 30
 +++
 
+## v0.8.0
+
+* Added support for connecting clusters that use the OVN (ovn-kubernetes) CNI plugin.
+* The active Gateway now performs periodic health checks on the connections to remote clusters,updates the Gateway
+  connection status, and adds latency statistics.
+* Gateways now export the following connection metrics which can be used with Prometheus. These are currently only supported
```

Comment:
I think that we should mention that this is done using TCP port 8080 (https://github.com/submariner-io/submariner-website/pull/392)